specify/specify7
# Add version field to ages

Fixes #5423

⚠️ **Note:** This PR affects database migrations. See [migration testing instructions](https://specify.github.io/testing/pull_requests.html#prs-tagged-with-label-migration).

Absolute and Relative age tables were missing the version field and hence the error

**DEV NOTE:**
We really should have a base model class which contains all common fields like version, timestamps etc. and have all our models inherit that class so we don't have to worry about forgetting to add those fields when creating a new table.


### Checklist

      and self-explanatory (or properly documented)

### Testing instructions



- Use a geology db
- Create a CO record with an Absolute/Relative or go to an existing one
- Update any field and click Save
- [ ] Verify there is no error
